backend/api/Controllers/DocumentController.cs

```diff
@@ -73,20 +73,33 @@ public IActionResult GetDocumentTypes()
         [SwaggerOperation(Tags = new[] { "documents" })]
         public IActionResult GetDocumentList()
         {
-            var result = _documentService.GetDocumentList();
+            var result = _documentService.GetStorageDocumentList();
             return new JsonResult(result);
         }
 
         /// <summary>
         /// Downloads the file for the correspoding file and document id.
         /// </summary>
         [HttpGet("{documentId}/files/{fileId}/download")]
-        [HasPermission(Permissions.PropertyAdd)]
+        //[HasPermission(Permissions.PropertyAdd)]
+        [ProducesResponseType(typeof(ExternalResult<FileDownload>), 200)]
+        [SwaggerOperation(Tags = new[] { "documents" })]
+        public async Task<IActionResult> DownloadFile(int documentId, int fileId)
```

Comment:
is int correct here? I just now that a lot of our ids are longs in other controllers/services.

Reply:
That is true, I will fix that on the next PR going over the types for the mayan interface

Reply:
agree with Devin - ids should be typed as **long**

Reply:
Hmm, when I first read the fileId I think I assumed it was the Id of a pims file, but it appears to by the id of the file in mayan. Not sure if that is worth a rename or not, up to you.

Reply:
I believe we should rename it so that is clear that the id is from mayan or pims

Reply:
Will update it in the a different pr for the next part of the document features